beta/project-griffon/set-up-project-griffon.md

```diff
@@ -131,6 +194,18 @@ func application(_ app: UIApplication, open url: URL, options: [UIApplication.Op
 You can end a session in the app interface by pressing the floating indicator and selecting **Disconnect**. You can also programmatically close an active session by using the following API.
 
 {% tabs %}
+{% tab title="Android" %}
+### endSession
+
+This API ends the active session and will ensure that no data is sent to a Project Griffon session.
```

Comment:
please remove "will" and change "ensure" to "ensures"

Reply:
Done.